contracts/Factory.cairo

```diff
@@ -180,16 +192,19 @@ func create_pair{
 
     let (salt) = hash2{hash_ptr=pedersen_ptr}(token0, token1);
 
+    let (fee_to_setter) = get_fee_to_setter();
+
     let constructor_calldata: felt* = alloc();
 
-    assert [constructor_calldata] = token0;
-    assert [constructor_calldata + 1] = token1;
-    // assert [constructor_calldata + 2] = contract_address
+    assert [constructor_calldata] = class_hash;
```

Comment:
Can rename this var to `pair_impl_class_hash` to be a bit more explicit about the intent of the code.